### ISSUE TITLE
Enabling Read Color Buffers leads to graphic artifacts in GT6

### ISSUE THREAD

**Issue by PY3AN** [RPCS3/rpcs3#9536]
Enabling Read Color Buffers leads to many graphic artifacts in GT6 but it can remove many another grafical issues. 

https://youtu.be/NZjYsB5WFWM

The issues that remove by Enabling Read Color Buffers:
The screen starts blinking when viewing the car in the gallery or while driving on the track

https://youtu.be/iPxy-8ySDEk

The issues that remove by Enabling Read Color Buffers and Write Color Buffers:

Gran Turismo 6 Mirror Issue (mixed with map) #8546;

Black screen when rendering 3D graphics in GT 6 version 1.06 and higher;

![Black screen in 3D on the track](https://user-images.githubusercontent.com/76910089/103486289-e21c9f00-4e0d-11eb-906f-fd09dd17fe3b.png)

[Black screen in 3D.log](https://github.com/RPCS3/rpcs3/files/5762085/Black.screen.in.3D.log)

After Enabling Read Color Buffers and Write Color Buffers decreases performance and the emulator starts to crash frequently in the menu with an error: 
F {PPU[0x1000002] Thread (_gcm_intr_thread) [0x00bc0a44]} VM: Access violation reading location 0x0 (unmapped memory) [type=u32]

[GT6_error1.log](https://github.com/RPCS3/rpcs3/files/5762118/GT6_error1.log)

Or 

F {RSX [0x0cc363c]} SIG: Thread terminated due to fatal error: [index buffer] Working buffer not big enough, buffer_length=16777216 allocated=320256 requested=4294967290 guard=65536 largest_pool=6144
(in file D:\a\1\s\rpcs3\Emu\RSX\Common\ring_buffer_helper.h:92[:24], in function alloc)

[GT6_error2.log](https://github.com/RPCS3/rpcs3/files/5762107/GT6_error2.log)

System configuration:
* OS Windows 10 20H2, 19042.685
* CPU AMD Ryzen 5 3600
* GPU AMD Radeon RX 580 8G
* Memory Crucial Ballistix Sport 16G
* Driver version 2.0.168

**Comment by benderscruffy** [comment]
is this still an issue ??

**Comment by Satan86** [comment]
Yes.

**Comment by benderscruffy** [comment]
thanks

**Comment by digant73** [comment]
@Satan86 is that still an issue?

**Comment by Satan86** [comment]
yeah but this can probably be closed as i have opened #12567 which encapsulates this issue and talks about some others in gt6 too
